sdk/eventhub/Azure.Messaging.EventHubs/src/Azure.Messaging.EventHubs.csproj

```diff
@@ -13,6 +13,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Azure.Core.Experimental" />
```

Comment:
I looked at what other SDKs using Experimental are doing, and they do a package reference. So, I did the same here.

Reply:
The latest released Experimental package has BinaryData in it, which is the only thing we use from Experimental. ObjectSerializer is already released. So, no version is needed. Also, I've added a comment to the package reference to indicate that it is only used for beta releases.